ngraph/frontend/paddlepaddle/src/op/hard_sigmoid.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "hard_sigmoid.hpp"
+#include <ngraph/builder/make_constant.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs hard_sigmoid(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("X");
+                    auto dtype = data.get_element_type();
+                    float slope = node.get_attribute<float>("slope", 0.2f);
+                    float offset = node.get_attribute<float>("offset", 0.5f);
+                    auto alpha = builder::make_constant(dtype, {}, slope);
```

Comment:
please don't use builder API, it creates broadcast nodes and you don't need it in this case.